dask_kubernetes/core.py

```diff
@@ -388,19 +386,19 @@ def __init__(
         security=None,
         scheduler_service_wait_timeout=None,
         scheduler_pod_template=None,
-        **kwargs
+        **kwargs,
     ):
         self.pod_template = pod_template
         self.scheduler_pod_template = scheduler_pod_template
-        self._generate_name = name
-        self._namespace = namespace
-        self._n_workers = n_workers
-        self._idle_timeout = idle_timeout
-        self._deploy_mode = deploy_mode
-        self._protocol = protocol
-        self._interface = interface
-        self._dashboard_address = dashboard_address
-        self._scheduler_service_wait_timeout = scheduler_service_wait_timeout
+        self.__generate_name = name
+        self.__namespace = namespace
+        self.n_workers = n_workers
+        self.idle_timeout = idle_timeout
+        self.deploy_mode = deploy_mode
+        self.protocol = protocol
+        self.interface = interface
+        self.dashboard_address = dashboard_address
+        self.scheduler_service_wait_timeout = scheduler_service_wait_timeout
```

Comment:
You've made a number of private attributes public here, what is the rationale behind this?

Reply:
I find it is good practice to leave __init__ kwargs "as they are". Trying to hide anything away from the end user doesn't work in python anyway... Of course we can make these `_` or `__` if you prefer, but generally users expect `MyClass(key=value).key` to return the same value.